<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="swapnilband-w" ignored="0" name="Default test" tests="4" failures="1" timestamp="16 Jul 2018 10:42:37 GMT" time="0.142" errors="0">
  <testcase name="@AfterMethod quit" time="0.0" classname="Flipkart.flipTest.TestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Flipkart.flipTest.TestCases.quit(TestCases.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod quit -->
  <testcase name="@BeforeMethod FlipExecute" time="0.0" classname="Flipkart.flipTest.TestCases">
    <skipped/>
  </testcase> <!-- @BeforeMethod FlipExecute -->
  <testcase name="@AfterMethod quit" time="0.0" classname="Flipkart.flipTest.TestCases">
    <skipped/>
  </testcase> <!-- @AfterMethod quit -->
  <testcase name="@BeforeMethod FlipExecute" time="0.0" classname="Flipkart.flipTest.TestCases">
    <skipped/>
  </testcase> <!-- @BeforeMethod FlipExecute -->
  <testcase name="@AfterMethod quit" time="0.0" classname="Flipkart.flipTest.TestCases">
    <skipped/>
  </testcase> <!-- @AfterMethod quit -->
  <testcase name="@BeforeMethod FlipExecute" time="0.0" classname="Flipkart.flipTest.TestCases">
    <skipped/>
  </testcase> <!-- @BeforeMethod FlipExecute -->
  <testcase name="@AfterMethod quit" time="0.0" classname="Flipkart.flipTest.TestCases">
    <skipped/>
  </testcase> <!-- @AfterMethod quit -->
  <testcase name="Testcase" time="0.004" classname="Flipkart.flipTest.TestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.Flipkart.TestBase.FilpTestBase.takeScreenshot(FilpTestBase.java:331)
at com.Flipkart.TestBase.FilpTestBase.fn_LaunchBrowser(FilpTestBase.java:68)
at Flipkart.flipTest.TestCases.Testcase(TestCases.java:27)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Testcase -->
  <testcase name="Testcase" time="0.0" classname="Flipkart.flipTest.TestCases">
    <skipped/>
  </testcase> <!-- Testcase -->
  <testcase name="Testcase" time="0.0" classname="Flipkart.flipTest.TestCases">
    <skipped/>
  </testcase> <!-- Testcase -->
  <testcase name="Testcase" time="0.0" classname="Flipkart.flipTest.TestCases">
    <skipped/>
  </testcase> <!-- Testcase -->
</testsuite> <!-- Default test -->
